examples/js/controls/VRControls.js

```diff
@@ -9,22 +9,28 @@ THREE.VRControls = function ( object, onError ) {
 
 	var vrInputs = [];
 
-	function filterInvalidDevices( devices ) {
+function filterInvalidDevices( devices ) {
 
 		// Exclude Cardboard position sensor if Oculus exists.
 
 		var oculusDevices = devices.filter( function ( device ) {
-
-			return device.deviceName.toLowerCase().indexOf( 'oculus' ) !== - 1;
+			if(device.deviceName) {
+				return device.deviceName.toLowerCase().indexOf( 'oculus' ) !== - 1;
+			} else {
+				return [];
```

Comment:
Later on, we test device.length

devices = filterInvalidDevices( devices );
for ( var i = 0; i < devices.length; i ++ ) {